tests/test_object.py

```diff
@@ -1718,3 +1719,13 @@ def test_iter(self):
             iter,
             Object(self.prog, "int []", address=0),
         )
+
+    def test__repr_pretty(self):
+        obj = Object(self.prog, "int", value=0)
+        pretty_printer_mock = unittest.mock.Mock()
```

Comment:
I hesitate a bit about using the Mock class from unittest as I don't see Mock being uses elsewhere, not sure if there's a better alternative.

Reply:
I think Mock makes sense for this test.